Shebang at top of ipythonPlugin should be /usr/bin/env python #3929

kpaharelau opened the pull request:
I think that using env (https://en.wikipedia.org/wiki/Env) command is a good idea. I have changed all start scripts for python plugins

kpaharelau committed:
Shebang at top of ipythonPlugin should be /usr/bin/env python #3929

I think that using env (https://en.wikipedia.org/wiki/Env) command is a good idea. I have changed all start scripts for python plugins

kpaharelau commented on plugin/ipythonPlugins/src/dist/ipython/ipythonPlugin:
```diff
@@ -45,4 +45,4 @@ os.environ["PYTHONPATH"] = plugin_dir + old_path
     subprocess.call(["python", plugin_dir+"/ipythonPluginStart.py"], shell=True)
 else:
-    os.execlp("python", "python", plugin_dir+"/ipythonPluginStart.py")
+    os.execlp("/usr/bin/env", "python", "python", plugin_dir+"/ipythonPluginStart.py")
```
yes

scottdraves replied:
that's not the shebang i don't see that it would do anything.
it's the 1st line of the file: https://en.wikipedia.org/wiki/Shebang_(Unix)

kpaharelau committed:
Shebang at top of ipythonPlugin should be /usr/bin/env python #3929

I have restored all old fumctionality. I have changed shebang only for ipythonPlugin to the '#!/usr/bin/env python'

kpaharelau committed:
Shebang at top of ipythonPlugin should be /usr/bin/env python #3929

spot [4:58 PM]
please do it for all the python-style plugins

DONE

kpaharelau committed:
Shebang at top of ipythonPlugin should be /usr/bin/env python #3929

spot [5:18 PM]
and remove the other changes -- a space is adding extra lines to the diff

DONE
